openlibrary/core/booknotes.py

```diff
@@ -5,7 +5,7 @@
 
 class Booknotes(db.CommonExtras):
     TABLENAME = "booknotes"
-    PRIMARY_KEY = ["username", "work_id", "edition_id"]
+    PRIMARY_KEY: tuple[str, ...] = ("username", "work_id", "edition_id")
```

Comment:
We shouldn't need to specify the type of these tuples ; it will be auto-deduced. Let's remove it from here and elsewhere where we specify the tuple type.
```suggestion
    PRIMARY_KEY = ("username", "work_id", "edition_id")
```

Reply:
I made changes by removing the tuple type decalration, and now there are no errors. However, in the file `openlibrary/plugins/openlibrary/tests/test_lists.py`, defining the tuple is necessary; otherwise,throws a type error.
![Screenshot from 2025-01-23 14-12-39](https://github.com/user-attachments/assets/3cd5ec37-ca80-47a6-91c0-dbd8892f3913)
so i declare the `tuple` type